pkg/servicescm/servicescm_test.go

```diff
@@ -0,0 +1,463 @@
+package servicescm
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParse(t *testing.T) {
```

Comment:
I just realized that there is no test that ensures the output of Generate() passes through Parse(). Won't that be a useful unit test? Otherwise I think we would only catch that during e2e testing.